m2e-maven-runtime/org.eclipse.m2e.maven.runtime/pom.xml

```diff
@@ -19,14 +19,14 @@
 	</parent>
 
 	<artifactId>org.eclipse.m2e.maven.runtime</artifactId>
-	<version>1.18.1-SNAPSHOT</version>
+	<version>1.19.0-SNAPSHOT</version>
 	<packaging>eclipse-plugin</packaging>
 
 	<name>M2E Embedded Maven Runtime (includes Incubating components)</name>
 
 	<properties>
 		<!-- maven core version -->
-		<maven-core.version>3.8.1</maven-core.version>
+		<maven-core.version>3.8.4</maven-core.version>
```

Comment:
The value of the `maven-resolver.version` property has to be set to `1.6.3` as well.